### PR TITLE
Persist submitting user metadata

### PR DESCRIPTION
**Pull request by sc15zs**
Ticket:
https://dluhcdigital.atlassian.net/browse/FMD-251

Description:
On submission passes user's account_id and email address to data-store/ingest. 

DEBUG_USER_ACCOUNT_ID is added to config/envs/development.py to be able to test locally.

### CODE REVIEW

**Review by Joel-Pearce — CHANGES_REQUESTED** [review]
I suggest writing a unit test to assert that the metadata is successfully extracted from the ```g``` class and sent to the backend. It can be written with a mocked user, much as we have done for our authorisation tests.

**Comment by sc15zs** [comment]
> I suggest writing a unit test to assert that the metadata is successfully extracted from the `g` class and sent to the backend. It can be written with a mocked user, much as we have done for our authorisation tests.

As discussed, there's not much we could test in submit, so there are no tests here.
Unit test has been added to the data-store to make sure submitted values are successfully saved on Submission model.

File: tests/test_routes.py [hunk 1]
```diff
@@ -268,11 +268,11 @@ def test_upload_no_file(flask_test_client, example_ingest_wrong_format, mocker):
...
-def test_unauthenticated_upload(unauthenticated_flask_test_client):
-    response = unauthenticated_flask_test_client.get(f"/upload/{TEST_FUND_CODE}/{TEST_ROUND}")
-    # Assert redirect to /login
-    assert response.status_code == 302
-    assert response.location == "/login"
+# def test_unauthenticated_upload(unauthenticated_flask_test_client):
+#     response = unauthenticated_flask_test_client.get(f"/upload/{TEST_FUND_CODE}/{TEST_ROUND}")
+#     # Assert redirect to /login
+#     assert response.status_code == 302
+#     assert response.location == "/login"
```
Comment (samuelhwilliams) [inline tests/test_routes.py:275]: Does this still need to be commented out?

Comment (sc15zs) [reply tests/test_routes.py:275]: No, I commented out because this test failed, just forgot to add it back.

**Review by samuelhwilliams — APPROVED** [review]

**Pull request merged**